Modules/Microsoft365DSC/DSCResources/MSFT_IntuneMobileThreatDefenseConnector/MSFT_IntuneMobileThreatDefenseConnector.mof

```diff
@@ -0,0 +1,31 @@
+[ClassVersion("1.0.0.0"), FriendlyName("IntuneMobileThreatDefenseConnector")]
+class MSFT_IntuneMobileThreatDefenseConnector : OMI_BaseResource
+{
+    [Key, Description("The unique identifier for an entity. Read-only.")] String Id;
+    [Write, Description("The unique identifier for an entity. Read-only.")] String DisplayName;
```

Comment:
DisplayName should be Key instead of ID. ID are unique to environments and Keys should always be human readable so that we can clone instances across tenants.

Reply:
This is another one where the Graph API, does not return the DisplayName. I just saw the service code where they are hardcoding the names. So I created a helper function which contains the hardcoded names: Get-MobileThreatDefenseConnectorIdOrDisplayName

Do you think it is still valid to make DisplayName as primary key?